src/pg_to_evalscript/javascript_processes/reduce_dimension.js

```diff
@@ -0,0 +1,31 @@
+function reduce_dimension(arguments) {
+  const { data, dimension, reducer } = arguments;
+
+  validateParameter({
+    processName: "reduce_dimension",
+    parameterName: "data",
+    value: data,
+    nullable: false,
+    required: true,
+  });
+
+  validateParameter({
```

Comment:
Dimension can also be validated against type as it should be a `string`.

Reply:
was added